File: UINavigationExtensionDemo/Feature/Basic/Controllers/ViewController06_ShadowImage.m
//
//  ViewController06_ShadowImage.m
//  UINavigationExtensionDemo
//
//  Created by lidan on 2020/10/26.
//

#import <UINavigationExtension/UINavigationExtension.h>

#import "ViewController06_ShadowImage.h"

@interface ViewController06_ShadowImage ()

@end

@implementation ViewController06_ShadowImage

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.view.backgroundColor = [UIColor whiteColor];
}

- (UIColor *)ue_navigationBarBackgroundColor {
    return [UIColor whiteColor];
}

- (UIImage *)ue_shadowImage {
    return [UIImage imageNamed:@"NavigationBarShadowImage"];
}

@end

File: NXNavigationExtensionDemo/UIKit/Basic/ViewController06_ShadowImage.swift
//
//  ViewController06_ShadowImage.swift
//  NXNavigationExtensionDemo
//
//  Created by lidan on 2021/11/8.
//

import UIKit

class ViewController06_ShadowImage: BaseViewController {

}

extension ViewController06_ShadowImage {
    
    override var nx_shadowImage: UIImage? {
        return UIImage(named: "NavigationBarShadowImage")
    }
    
    override var nx_navigationBarBackgroundColor: UIColor? {
        return view.backgroundColor
    }

}
